docs/modules.html

```diff
@@ -120,6 +120,8 @@
 };
 ```
 
+**Note**: `include` and `open` are very different! The former literally copies over definitions into the new module for it to appropriate it. The latter simply refer to stuff in the `open`ed module's scope.
```

Comment:
> **Note**: `include` and `open` are very different! The former literally copies over definitions into the new module for it to appropriate it. The latter simply refer to stuff in the `open`ed module's scope.

Some suggestions for wording tweaks:
- `for it to appropriate it` -> `for it to appropriate`
- `the later simpler refer to stuff in` -> `the later simply allows unqualified access to values, types and constructors in`